### ISSUE TITLE
Update Scout2 

### ISSUE THREAD

**Issue by ghost** [SecurityFTW/cs-suite#73]
Scout2 is deprecated it is now https://github.com/nccgroup/ScoutSuite

Can this be updated?

**Comment by shivankar-madaan** [comment]
yes we have it planned for the future and PR's are highly appreciated
